website/views/security_incidents.py

```
@@ -0,0 +1,51 @@
+
+
+class SecurityIncidentCreateView(LoginRequiredMixin, StaffRequiredMixin, CreateView):
+    model = SecurityIncident
+    form_class = SecurityIncidentForm
+    template_name = "security/incidents/incident_form.html"
+    success_url = reverse_lazy("security_dashboard")
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["form_title"] = "Create Security Incident"
+        context["unique_editor_count"] = incident.history.values("changed_by").distinct().count()
```

Ruff (0.14.7)

Undefined name `incident`

(F821)